translator/context/context.go

```diff
@@ -114,7 +114,7 @@ func (ctx *Context) SetMode(mode string) {
 	case config.ModeOnPrem:
 		ctx.mode = config.ModeOnPrem
 	default:
-		panic(fmt.Sprintf("Invalid mode %s. Valid mode values are %s and %s.\n", mode, config.ModeEC2, config.ModeOnPrem))
+		panic(fmt.Sprintf("Invalid mode %s. Valid mode values are %s, and %s.\n", mode, config.ModeEC2, config.ModeOnPrem))
```

Comment:
This could be reverted

Reply:
The comma suits the statements so I guess we don't need to revert it though.